src/styles/product.scss

```
@@ -0,0 +1,98 @@
+.product {
+  margin-right: 48px;
+
+  &__container {
+    background: #fff;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    margin: 0;
+    width: 200px;
+    height: 406px;
+    box-sizing: border-box;
+  }
+  
+  &:nth-last-child(-n + 4) {
+    margin-top: 48px;
+    padding: 0;
+    margin-bottom: 73px;
+  }
+  
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  
+  &__descripion {
+    margin: 0 16px;
+  }
+  
+  .image {
+    margin-bottom: -4px;
+  }
+  
+  &-image {
+    margin: 0;
+    width: 160px;
+    padding: 32px 19px 40px;
+    height: 134px;
+  }
```

```suggestion
  &-image {
    margin: 0;
    width: 160px;
    height: 134px;
  }
```
Не потрібно цього div-контейнера iamge. Просто використовуй img з display: block.
Падінги також лишні. Скоріш за все, це пофіксить деякі тести